### PR TITLE
Adds black formatter checker to Travis

### LINKED ISSUE #22: check black in CI
`black` is recommended in the `contributing.md` but it is not automatically checked.

There should be a check added to the CI.

`black --check` will return 0 (success) when there are no changes to be made, and non-zero when changes would have been made.

### PR DESCRIPTION
**Pull request by Yagueteiro**
Modifies the Travis script to add Black checker.

Closes #22

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ python:
 install:
   - pip install -r requirements.txt
   - pip install typelint
+  - pip install black
 # command to run tests
 script:
   - python code/ingest.py || python3 code/ingest.py
   - python -m typelint --quiet code/ingest.py
+  - if !((python -m black --check)) ; then echo "Black formatter is required!"; fi
```
Comment (drj11) [inline .travis.yml:14]: you don't need the `if` or the `echo`, you can literally run `python -m black --check` and then the Travis CI build will fail when this fails.

**Review by drj11 — CHANGES_REQUESTED** [review]
this is great, but can you please simplify the command as i've suggested? Thanks!

File: .travis.yml (hunk 1 above)
Comment (Yagueteiro) [reply .travis.yml]: Done!

**Comment by drj11** [comment]
thanks!

**Pull request merged**